### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by jonathanarde**
1. updating the landing page based on google docs
2. change css styles of all pages using bootstrap container

### CODE REVIEW

File: views/footer.jade [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.container-fluid(style='padding-bottom: 20px')
+    footer
+        .row
+            .col-xs-6
+                h5 USING SLACK
```
Comment (chrislim) [inline views/footer.jade]: change this before committing it.

File: views/footer.jade [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                a About/FAQ
```
Comment (chrislim) [inline views/footer.jade]: link these to placeholder pages before committing.

File: views/index.jade [hunk 3]
```diff
@@ -1,128 +1,126 @@
-extends layout
+doctype html
+html
+    head
+        title= title
+        link(rel='stylesheet', href='/stylesheets/style.css')
+        link(rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css")
+        link(rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css")
```
Comment (chrislim) [inline views/index.jade]: what do we use jquery ui for?

File: views/index.jade [hunk 4]
```diff
@@ -1,128 +1,126 @@
...
+        link(rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.11.2/css/bootstrap-select.min.css")
+        link(href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet")
```
Comment (chrislim) [inline views/index.jade]: why are the glyphicons from a different version than the bootstrap css?
I recommend loading this from your bower_components instead or something like it. No need to use other people's cdn for it.

File: views/index.jade [hunk 5]
```diff
@@ -1,128 +1,126 @@
...
+        link(rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" 
+        integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" 
+        crossorigin="anonymous")
+        link(rel='stylesheet' href='/stylesheets/slate.min.css')
+        link(rel="icon" href="/images/glorifyicon_32x32.png")
+    body
+        include importJS
+        //- img#heroImage(src='/images/hero_image.jpg')
```
Comment (chrislim) [inline views/index.jade]: cleanup

File: views/index.jade [hunk 6]
```diff
@@ -1,128 +1,126 @@
...
+        nav.navbar.navbar-default.navbar-fixed-top(style='margin-bottom: 0px; border: none')
+            .container
+                .navbar-header
+                    button.navbar-toggle.collapsed(type="button" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false")
```
Comment (chrislim) [inline views/index.jade]: i think you want to use a different id rather than copying the one from the bootstrap docs.

File: views/index.jade [hunk 7]
```diff
@@ -1,128 +1,126 @@
...
 
-            #bs-example-navbar-collapse-1.collapse.navbar-collapse
-                ul.nav.navbar-nav
-                    - if(login && user.username !== 'admin')
-                        li
-                            a(href="/user/library") Library
-                        li
-                            a(href="/user/playlist") Playlist
+        .container
+            #anyLanguage
```
Comment (chrislim) [inline views/index.jade]: is this id used somewhere? if not we can drop it.

File: views/index.jade [hunk 8]
```diff
@@ -1,128 +1,126 @@
...
-                            a(href="/user/signup") Sign Up
+            #features
+                .row
+                    .col-xs-12.col-sm-6.col-md-6
+                        .thumbnail(style='text-align:center')
+                            h4 Easily Export Slides
+                            .circularIcon(style='margin:auto')
+                                img(src='/images/desktop-computer.png' style='height: 40px; margin-top: 5px')
+                            .caption
+                                //- img.img-responsive(alt="create slides" src="images/create-feature-img.jpeg")
```
Comment (chrislim) [inline views/index.jade]: cleanup.

File: views/index.jade [hunk 9]
```diff
@@ -1,128 +1,126 @@
...
+                                p Use Glorify to get song lyrics in various languages, and contribute translations to enable 
+                                a.btn.btn-default(href='/discover') Find Songs
+                .row
+                    .col-xs-12.col-sm-6.col-md-6
+                        .thumbnail(style='text-align:center')
+                            h4 Share your songs with the Global Church
+                            .circularIcon(style='margin:auto')
+                                img(src='/images/business-organization.png' style='height: 40px; margin-top: 5px')
+                            .caption
+                                //- img.img-responsive(alt="create slides" src="images/create-feature-img.jpeg")
```
Comment (chrislim) [inline views/index.jade]: cleanup

File: views/index.jade [hunk 10]
```diff
@@ -1,128 +1,126 @@
...
+                                //- img.img-responsive(alt="create slides" src="images/create-feature-img.jpeg")    
+                                p Songwriters can share their worship songs with the Global Church on Glorify so that even more believers can worship freely.
+                                //- button.btn.btn-default Create Slides   
+                    .col-xs-12.col-sm-6.col-md-6
+                        .thumbnail(style='text-align:center')
+                            h4 Free for all to worship
+                            .circularIcon(style='margin:auto')
+                                img(src='/images/broken-link.png' style='height: 40px; margin-top: 5px')
+                            .caption
+                                //- img.img-responsive(alt="create slides" src="images/create-feature-img.jpeg")
```
Comment (chrislim) [inline views/index.jade]: cleanup

File: views/index.jade [hunk 11]
```diff
@@ -1,128 +1,126 @@
...
+                            h4 Share your songs with the Global Church
+                            .circularIcon(style='margin:auto')
+                                img(src='/images/business-organization.png' style='height: 40px; margin-top: 5px')
+                            .caption
+                                //- img.img-responsive(alt="create slides" src="images/create-feature-img.jpeg")    
+                                p Songwriters can share their worship songs with the Global Church on Glorify so that even more believers can worship freely.
+                                //- button.btn.btn-default Create Slides   
+                    .col-xs-12.col-sm-6.col-md-6
+                        .thumbnail(style='text-align:center')
+                            h4 Free for all to worship
```
Comment (chrislim) [inline views/index.jade]: "Freedom to worship" instead of free for all?

File: views/index.jade [hunk 12]
```diff
@@ -1,128 +1,126 @@
...
 
-    hr
-    .container-fluid
-    h1(style='text-align:center; margin-top: 0;') Worship Freely
-    h4(style='text-align:center; font-style:italic;') After this I looked, and behold, a great multitude that no one could number, from every nation, from all tribes and peoples and languages, standing before the throne and before the Lamb, clothed in white robes, with palm branches in their hands, and crying out with a loud voice, “Salvation belongs to our God who sits on the throne, and to the Lamb!” - Revelation 7:9-11 ESV
-    hr
+                p.
+                    We want to see the Church unleashed from the status quo. We want to see the Church united in worship across language and culture. We want to see the Church be able to worship freely. That's why we built Glorify.cc.
```
Comment (chrislim) [inline views/index.jade]: Drop the first sentence about the status quo, or say
"We want to see the Church unleashed from the segregated status quo."

File: views/topnav.jade [hunk 13]
```diff
@@ -24,7 +25,7 @@ nav.navbar.navbar-default
                 li
                     a(href='/help') Help
             .navbar-form.navbar-left
-                #searchBox
+                #searchBox(style='width: 330px')
```
Comment (chrislim) [inline views/topnav.jade]: why is this width hardcoded? Could it be problematic for mobile devices?

Comment (jonathanarde) [inline views/topnav.jade]: it wont, i have tested it

**Pull request merged**